src/test/java/com/microsoft/sqlserver/jdbc/KerberosTest.java

```diff
@@ -33,7 +34,7 @@ public void testUseDefaultJaasConfigConnectionStringPropertyTrue() throws Except
 
         // Initial connection should succeed with default JAAS config
         try (SQLServerConnection conn = (SQLServerConnection) DriverManager.getConnection(connectionStringUseDefaultJaasConfig)) {
-            ResultSet rs = conn.createStatement().executeQuery("select auth_scheme from sys.dm_exec_connections where session_id=@@spid");
+            ResultSet rs = conn.createStatement().executeQuery(authSchemeQuery);
             rs.next();
             Assertions.assertEquals(kerberosAuth, rs.getString(1));
```

Comment:
nitpick - can we just import this as in other tests so we don't have to prefix each time?

Reply:
Done.